### PR TITLE
Add configs/id/services API. Add posture check role attributes list API. Fixes #120. Fixes #121

### LINKED ISSUE #120: Add API for retrieving services referencing a config
Add API for retrieving services referencing a config

### LINKED ISSUE #121: Add API for retrieving the set of attribute roles used by posture checks
Add API for retreiving the set of attribute roles used by posture checks

### PR DESCRIPTION
**Pull request by plorenz**

### CODE REVIEW

File: source/management/configs.yml [hunk 1]
```diff
@@ -150,6 +150,32 @@ paths:
...
 
+  configs-id-services:
+    parameters:
+      - $ref: '../shared/parameters.yml#/id'
+    get:
+      summary: List services referenced by a config
+      description: |
+        Retrieves a list of service resources that reference a given config; supports filtering, sorting, and pagination. Requires admin access.
+      security:
+        - ztSession: [ ]
```
Comment (andrewpmartinez) [inline source/management/configs.yml:161]: Missing OIDC support

```
      security:
        - ztSession: []
        - oauth2: [openid]
```

Comment (andrewpmartinez) [reply source/management/configs.yml:161]: If this was copied from somewhere as a base, it is also missing it.

**Review by andrewpmartinez — DISMISSED** [review]
OIDC support. Also missing a version bump in /source/management.yml and /source/client.yml

File: source/management/configs.yml (hunk 1 above)
Comment (plorenz) [reply source/management/configs.yml:161]: went through and fixed all the missing oauth2 bits. Bumped the version

**Review by andrewpmartinez — APPROVED** [review]

**Pull request merged**